2019/11xxx/CVE-2019-11040.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
CNA(s) of user (mitre|dwf|adobe|airbus|alibaba|google_android|apache|apple|asustor|atlassian|autodesk|blackberry|bah|brocade|canonical|ca|certcc|checkpoint|cisco|dahua|debian|dell|drupal|duo|eclipse|elastic|f5|flexera|forcepoint|fortinet|freebsd|google_chrome|hackerone|hpe|hp|huawei|ibm|icscert|intel|isc|ioactive|juniper|jpcert|kaspersky|krcert|larry_cashdollar|lenovo|marklogic|mcafee|microfocus|microsoft|mozilla|netapp|netflix|netgear|nodejs|nvidia|obdev|openssl|oracle|puppet|qihoo|qnap|qualcomm|rapid7|redhat|riverbed|sap|schneider|siemens|sgi|symantec|synology|talos|tenable|tibco|trendmicro|vmware|yandex|zephyr|zdi|zte|cybersecurity_malaysia|facebook|hikvision|cloudflare|palo_alto|hillstone|sonicwall|naver|avaya|odoo|cspcert|appthority|twcert|mongodb|jci|suse|snyk|jenkins|kubernetes|fedora|joshbressers|) do not match owning CNA.